util/shared/typescript.mjs

```diff
@@ -386,7 +386,8 @@ function getSchemaShape(moduleJson = {}, json = {}, schemas = {}, name = '', opt
       }
       else {
         // grab the type for the non-array schema, so we get the link for free
-        let type = getSchemaType(module, json.items, schemas, {code: options.code, link: options.link, title: options.title})
+        // added asPath and baseUrl to generate proper URL
```

Comment:
What does this have to do with `asPath` and `baseUrl`?

Reply:
I think you're saying those get passed in and used by `getSchemaType` because they are part of the `options` hash, right?

Reply:
Yes .